tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -305,7 +306,7 @@ export class RunsTableContainer implements OnInit, OnDestroy {
 
     const getFilteredItems$ = this.getFilteredItems$(
       this.allUnsortedRunTableItems$
-    ).pipe(shareReplay(1));
+    ).pipe(takeUntil(this.ngUnsubscribe), shareReplay(1));
```

Comment:
Do you mind putting a comma after shareReplay(1) here so it formats on multiple lines?

Reply:
I just tried, but unfortunately Prettier seems to undo the comma and remove the multiline, I think because `this.getFilteredItems$(this.allUnsortedRunTableItems$)` did not fit onto one line.